RELEASING.adoc

```diff
@@ -2,20 +2,33 @@
 
 This document gives the steps required to perform a release of Kiali. This document is only useful to Kiali release managers.
 
-NOTE: There are two methods of producing release images here. The Kiali team will decide which one they want to use.
-
 The steps to perform a release are very simple and easy to execute thanks to the way link:.travis.yml[Travis] is setup.
 
-Before running these steps, it is assumed the link:https://github.com/kiali/kiali/tree/master[master] branch is all up to date and includes all code that is to be released and that your local `master` branch is up to date with this remote `master` branch.
+Before running these steps, it is assumed the link:https://github.com/kiali/kiali/tree/master[kiali master] branch and the link:https://github.com/kiali/kiali-ui/tree/master[kiali-ui master] branch are all up to date and include all code that is to be released and that your local `master` branches are up to date with the remote `master` branches.
+
+== Kiali-UI
+
+1) Tag the Kiali-UI master branch in this repo: https://github.com/kiali/kiali-ui
+
+```
+$ git tag v1.0.0
```

Comment:
Maybe add a hint somewhere that this version refers to main Kiali version, not the console version (it might be confusing since we're in the kiali-ui repo)

Reply:
Actually, this is supposed to be the kiali version - I'm trying to have the common kiali image version string so we know "this is the console for the kiali docker image version v1.0.0"

Reply:
I added a blurb in the README to explain that the kiali-ui is kinda weird and for now, we have to deal with it separately. In the future we will need to figure out how to more easily deal with the strange npm version numbers that are created based on the travis build numbers.

Reply:
I think we would need to have a release process for the ui as well.

Reply:
Hmm @josejulio do you mean a separate process ?
If we could converge to a single process and an unique version number it would be IMO easier to reason about

Reply:
not exactly separated, but we need to agree if we are going to release the ui in npm if so then we would to update the kiali-ui repo with the right version.